src/core/annotation.js

```diff
@@ -547,8 +548,8 @@ var TextWidgetAnnotation = (function TextWidgetAnnotationClosure() {
     WidgetAnnotation.call(this, params);
 
     this.data.textAlignment = Util.getInheritableProperty(params.dict, 'Q');
-    this.data.annotationType = AnnotationType.WIDGET;
```

Comment:
This change is no longer necessary when you rebase the patch as it has been fixed already in #6757.